.github/workflows/code-check.yml

```diff
@@ -21,6 +21,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     env:
       NX_CLOUD_ACCESS_TOKEN: ${{ secrets.NX_CLOUD_ACCESS_TOKEN }}
+      NODE_OPTIONS: "--max_old_space_size=4096"
```

Comment:
Is it needed for `yarn test` command only?
(if so, maybe it should be set only for the step `Test`)

Is it for both OS?
(if so, the set of this EnvVar can be conditional)

Reply:
updated for only `yarn test`, only Windows